### ISSUE TITLE
Can't train summarization model on my data

### ISSUE THREAD

**Issue by mainakchain** [tensorflow/tensor2tensor#873]
### Description
I created a new summarizing problem with a new dataset of my own. I followed [new problem](https://tensorflow.github.io/tensor2tensor/new_problem.html) with [training walkthrough](https://tensorflow.github.io/tensor2tensor/walkthrough.html), but whenever I start training all we get in the train directory are:

events.out.tfevents.*
flags.txt
flags_t2t.txt
graph.pbtxt
hparams.json
After that no model or checkpoint is saved. In other words, my problem is not training on the generated data using t2t-datagen. I even tried using --generate_data during t2t_trainer training. The data generation goes smoothly as:
`INFO:tensorflow:Generating data for summarize_cnn_dailymail32k
INFO:tensorflow:Not downloading, file already found: /home/mainak.chain/Wikisum_dev/my_t2t_datagen/all_train.txt
INFO:tensorflow:Generating vocab file: /home/mainak.chain/Wikisum_dev/my_data/vocab.cnndailymail.32768
INFO:tensorflow:Found 128 examples
INFO:tensorflow:Trying min_count 500
INFO:tensorflow:Iteration 0
INFO:tensorflow:vocab_size = 1841
INFO:tensorflow:Iteration 1
INFO:tensorflow:vocab_size = 878
INFO:tensorflow:Iteration 2
INFO:tensorflow:vocab_size = 953
INFO:tensorflow:Iteration 3
INFO:tensorflow:vocab_size = 938
INFO:tensorflow:Trying min_count 250
......
INFO:tensorflow:Trying min_count 1
INFO:tensorflow:Iteration 0
INFO:tensorflow:vocab_size = 84356
INFO:tensorflow:Iteration 1
INFO:tensorflow:vocab_size = 25813
INFO:tensorflow:Iteration 2
INFO:tensorflow:vocab_size = 25813
INFO:tensorflow:Iteration 3
INFO:tensorflow:vocab_size = 25813
INFO:tensorflow:Not downloading, file already found: /home/mainak.chain/Wikisum_dev/my_t2t_datagen/all_train.txt
INFO:tensorflow:Writing cnndm.train
INFO:tensorflow:Found 128 examples
INFO:tensorflow:Found 128 examples
INFO:tensorflow:Generated 128 Examples`
...

### Environment information
I am using virtualenv environment of python 3.4.3 and pip version 10.0.1 with : 

```
OS: <Ubuntu 14.04 LTS Mem: 7.7 GB, Intel® Core™ i5-4300U CPU @ 1.90GHz × 4 >

$ pip freeze | grep tensor
tensor2tensor==1.6.3
tensorboard==1.8.0
tensorflow==1.8.0

$ python -V
Python 3.4.3
```
I have been at this for some days. I am not able to train my summarization model. At first I created a new problem, and tried registering and training using it. Recently, I shaped my data in the summarize_cnn_dailymail32k problem format and tried to train it with the predefined cnn_dailymail problem. But, it never outputs more than the above 5 files. (When I train other problems, I happen to get files like checkpoint, model.ckpt.* sometime after my training starts using t2t-trainer, which is so obvious).

To my surprise, the problem training on my data (even with predefined daily mail problem) consumes all the cores of my system and even takes up most of my GPU RAM, without giving any info neither on the screen nor as output files. For getting a deeper understanding, I tried to put up the output_dir on tensorboard. All it showed was a graph of transformer architecture. The projector, scalar of other curves are not formed at all. The display logs I obtain at last in terminal are shown.
### Display Logs
`INFO:tensorflow:Transforming body output with symbol_modality_25813_16.top
WARNING:tensorflow:From /home/mainak.chain/env_wikisum/lib/python3.4/site-packages/tensor2tensor/layers/common_layers.py:1883: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See @{tf.nn.softmax_cross_entropy_with_logits_v2}.

INFO:tensorflow:Base learning rate: 2.000000
INFO:tensorflow:Trainable Variables Total size: 421168
INFO:tensorflow:Using optimizer Adam
/home/mainak.chain/env_wikisum/lib/python3.4/site-packages/tensorflow/python/ops/gradients_impl.py:100: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
2018-06-18 11:57:16.775278: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.`

**Comment by ericp96** [comment]
@mainakchain do you have any insight as to how you fixed this?
